pre_pick_rad: [0.0, 0.1, 0.50, 0.45, -1.8, -0.02]
post_pick_rad: [0.0, 0.3, 0.0, 0.4, -1.8, 0.0]
